docs/cn/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -1,232 +1,328 @@
 ---
 layout: global
-title: 在Docker上运行Alluxio
-nickname: 在Docker上运行Alluxio
+title: 在集群上独立运行Alluxio
+nickname: 在集群上独立运行Alluxio
 group: Install Alluxio
-priority: 3
+priority: 4
 ---
 
-* 内容列表
-{:toc}
-
-Alluxio可以运行在一个Docker容器中，本指南介绍如何使用Alluxio Github仓库提供的Dockerfile来将Alluxio运行在Docker之中。
 
-# 基础教程
+Docker可用于简化Alluxio服务器的部署和管理。使用 Dockerhub上的[alluxio/alluxio](https://hub.docker.com/r/alluxio/alluxio/) Docker镜像，只要运行几个docker run命令就可以从零到运行Alluxio集群。本文档提供了以本地磁盘作为底层存储的单个节点上运行Dockerized Alluxio的教程。我们也还将讨论更高级的主题以及如何解决遇到的问题。
+* 目录
+{:toc}
 
-这篇教程主要介绍如何在单节点上使用Docker运行Alluxio。
+## 前提条件
 
-# 前期准备
+- 一台安装了Docker主机。
+- 开放端口19998、19999、29998、29999和30000
 
-一台Linux主机。在该指南中，我们将使用一台全新的运行着Amazon Linux的EC2主机。主机容量不需要太大，这里我们使用t2.small。当为实例设置网络安全时，允许端口19998-19999和29998-30000通信。
+如果自己没有安装了Docker的主机，则可以在AWS上创建一个小AWS EC2计算实例（例如t2.small），以便随本教程一起学习。发放计算实例过程中对security group做出以下设置，以便以下端口对你的IP地址和Alluxio客户端（例如，远程Spark集群）CIDR范围内地址时开放的：
 
-# 启动一个独立模式集群
++ 对于Alluxio服务器和客户端CIDR地址范围内端口19998：允许客户端和workers与Alluxio Master RPC进程进行通信。
++ 你的浏览器的IP地址端口19999：允许您访问Alluxio master Web UI。
++ Alluxio和客户端的CIDR范围内端口29999：允许客户端与Alluxio Worker RPC进程进行通信。
++ 你的浏览器的IP地址端口30000：允许您访问Alluxio worker Web UI。
 
-你需要在你的Linux主机上执行以下的所有步骤。
 
-## 安装Docker
+在发放实例后（后续称为Docker Host）运行以下命令来设置Docker
 
 ```console
-$ sudo yum install -y docker git
+$ sudo yum install -y docker
 $ sudo service docker start
 # Add the current user to the docker group
 $ sudo usermod -a -G docker $(id -u -n)
 # Log out and log back in again to pick up the group changes
 $ exit
 ```
 
-## 复制Alluxio仓库
+## 准备Docker Volume存储数据
 
-```console
-$ git clone https://github.com/Alluxio/alluxio.git
-```
+默认情况下容器内创建的所有文件都存在可写的容器layer。当该容器不再存在时，数据不会保留。 [Docker volumes](https://docs.docker.com/storage/volumes/)是将数据保存到容器外部的方法。最常使用的以下两种的Docker volumes：
+
++ **Host Volume**：你来决定和管理在Docker 主机的文件系统中在哪里存储和共享容器数据。创建host volume，运行：
+
+  ```console
+  $ docker run -v /path/on/host:/path/in/container ...
+  ```
+  文件或目录通过Docker主机上的完整路径引用。可以是已经存在于Docker主机上的，如果尚不存在则会自动创建。
+
++ **Named Volume**：Docker来管理其位置。用特定名称来指代即可。创建命名卷，运行：
+
+  ```console
+  $ docker volume create volumeName
+  $ docker run -v  volumeName:/path/in/container ...
+  ```
+
+Alluxio容器用host volume或named volume都可以。出于测试目的话，建议使用host volume，因为它是最容易使用的volume，并且性能非常好。更重要的是，你知道在主机文件系统中所引用数据的位置，并且可以在容器外部直接和很容易的做针对文件的操作。
+
+因此，我们将使用host volume并将host目录 `/ alluxio_ufs`挂载到容器路径 `/ opt / alluxio / underFSStorage`，这是Alluxio docker镜像中Alluxio UFS root挂载点的默认设置：
```

Comment:
`/ alluxio_ufs` think those as code and remove the added extra space, they represent local filesystem paths which are not allowed extra spaces

Reply:
the same for the whole docs